internal/handlers/telegram/handler.go

```diff
@@ -64,8 +64,12 @@ documentHandler receives a document object from Telegram, and sends
 a notification to IRC.
 */
 func documentHandler(tg *Client, u *tgbotapi.Message) {
-
-	formatted := u.From.UserName + " shared a file (" + u.Document.MimeType + ")"
+	formatted := u.From.String() + " shared a file ("
+	if u.Document.MimeType != "" {
+		formatted += u.Document.MimeType + ")"
+	} else {
+		formatted += u.Document.FileID + ")"
```

Comment:
So this is actually the whole issue with #159 
We explicity do _not_ want the unfiltered link to the document download, as this [displays the bot api token](https://github.com/RITlug/teleirc/issues/241). For now, I would recommend only showing the `MimeType` if it is available, and ignore the `FileID` entirely. 

This is something I forgot about in the first review.

Reply:
I see your point. For security reason, we definitely should not display the api token.